### DISCUSSION TITLE
Popup does not close when the tabs are switched in management

### DISCUSSION THREAD

**Discussion by ackepenek** [rabbitmq/rabbitmq-server#10797] [Other]
### Describe the bug

![image](https://github.com/rabbitmq/rabbitmq-server/assets/1234305/3e580f16-d33a-4b9a-aa2f-e1eddf9b340e)
If open the popup and then change the tabs and screen then the popup does not close and the popup still appears in the screen.

### Reproduction steps

1.Open the management screen
2.Click the +/- (Click to change columns) to open popup
3.Change the tab (from Overview to Connections or from Exchanges to Channels) (without click the close button in the popup)
4.The popup thats related with previous scope is still appear in the screen

### Expected behavior

If the popup is opened and than changed the tab or section without click the close button in the popup then the popup should be closed.

**Comment by michaelklishin** [comment]
It can, nonetheless, be closed easily before switching tabs. You are welcome to submit a PR that would close the pane when a tab is switched. I'm afraid this won't be a priority for the small RabbitMQ team.

**Reply by ackepenek** [reply]
I already opened the PR :) It's pleasure to contribute to the team
https://github.com/rabbitmq/rabbitmq-server/pull/10799
